### DISCUSSION TITLE
Pybricks code editor as an electon app

### DISCUSSION THREAD

**Discussion by mrinalraghupathi** [pybricks/support#1417] [Ideas]
This might be overkill, but I was curious. What would need to be done to bundle the pybricks code editor as a standalone electron app? I see some advantages to doing this for kids being able to work directly with the files they save on their computer.

**Comment by dlech** [comment]
Can you describe more about what you would like to do with the files on your computer than you can't currently do with Pybricks Code?

**Reply by mrinalraghupathi** [reply]
Mostly it's functionality to autosave have a way of navigating/saving files that's a little closer to a code editor like VS code. Currently the way it works (at least when I install pybricks code) is that you have to download and import files. and downloading a set of files saves a zip file. It may be overkill to deploy this as an electron app, but I've had kids who are look for something a little more like a code editor, where the files are on the drive and cmd/ctrl-s will save to the drive etc...

I've also really enjoyed using pybricks with VS code (thanks for adding those instructions, I hadn't seen them last time I was using pybricks!) I suppose that *is* an electron app which handles this use case :)!
